mlfinlab/bet_sizing/targeting.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains functionality for determining bet sizes for investments based on volatility targeting.
+"""
+
+import numpy as np
+
+
+def volatility_targeting(returns: np.ndarray, target_vol: float, annualize_factor: int):
+    """
+    Calculate the position size to make the expected volatility of the portfolio be the target variability.
+
+    :param returns: (np.ndarray) This is series of returns of your strategy or a asset.
+    :param target_vol: (float) volatility(%) that you target.
+    :return: (float) position size you should use to change the volatility to be target volatility.
```

Comment:
Please change the first letters of discription to capital ones for style uniformity, like so:

> :param target_vol: (float) Volatility ...

>  :return: (float) Position ...

Also, please use passive voice for variable description, like:

> Series of returns of a strategy/asset.

> Target volatility as a percentage. 

> Position size used to change the volatility to be target volatility.